ocfweb/error404/er404.py

```diff
@@ -0,0 +1,9 @@
+from django.shortcuts import render
+
+
+# Function for rendering the error 404 page
+def error404(request):
+    return render(
+            request,
+            '404.html')
```

Comment:
probably better to just write this all on one line? `return render(request, '404.html')`